.travis.yml

```diff
@@ -26,7 +26,7 @@ cache:
 before_install:
 - echo $TRAVIS_OS_NAME
 - export CSL_SYSTEM_TAG=$TRAVIS_OS_NAME
-- export DCONFIG=travis
+- export DCONFIG=ci_wallet
```

Comment:
No, the idea is that it's always `ci_wallet`. If we want to change it from `qanet` to `testnet` (or whatever else) it should be done in `constants.yaml`.
`ci` is just an alias for one of existing modes.